easypost/services/insurance_service.py

```diff
@@ -47,3 +52,13 @@ def get_next_page(
             params.update(optional_params)
 
         return self.all(**params)
+
+    def refund(self, id: str) -> Insurance:
+        url = f"/insurances/{id}/refund"
+        response = Requestor(self._client).request(
+            method=RequestMethod.POST,
+            url=url,
+            beta=True,  # Remove beta when endpoint is in GA.
```

Comment:
```suggestion
            beta=True,  # TODO: Remove beta when endpoint is in GA.
```

Because this requires follow-up. we should add `TODO` to ensure we differentiate this from a normal code comment meant to explain something

Reply:
Good catch, I forgot to add it. This diff will not be landed until the endpoint is in GA. Marking this PR back to draft